src/basic/proc-cmdline.c

```diff
@@ -392,13 +392,13 @@ int proc_cmdline_get_key(const char *key, ProcCmdlineFlags flags, char **ret_val
         return cmdline_get_key(args, key, flags, ret_value);
 }
 
-int proc_cmdline_get_bool(const char *key, bool *ret) {
+int proc_cmdline_get_bool(const char *key, ProcCmdlineFlags flags, bool *ret) {
```

Comment:
`battery-check` can make use of this too. But that's out of the scope for this PR so can be done later.